File: trendify.Client/src/app/models/order/orderSummary-model.ts
export class OrderSummaryModel {
    id!: number;
    orderNumber!: string;
    orderDate!: string;
    total!: number;
    status!: string;

    constructor(init?: Partial<OrderSummaryModel>) {
        if (init) {
            Object.assign(this, init);
        }
    }
}

File: trendify.Server/trendify.Core/Models/Orders/OrderSummaryModel.cs
namespace trendify.Core.Models.Orders
{
    public class OrderSummaryModel
    {
        public int Id { get; set; }
        public string OrderNumber { get; set; } = null!;
        public DateTimeOffset OrderDate { get; set; }
        public decimal Total { get; set; }
        public string Status { get; set; } = null!;
    }
}
